### DISCUSSION TITLE
How to get a tag "way" based on "nd ref" search?

### DISCUSSION THREAD

**Discussion by lhcsnelm** [osm-search/Nominatim#2834] [Ideas]
Hi

I am trying to get a chink of the tag based on element search.  Nominatim has such query command? 

example)
Get <way 12345678> ~ </way> based on "nd ref" keyword 321321321

<way id="12345678" ... >
  <nd ref="321321321"/>
  <nd ref="421321321"/>
  ...
  <tag k="highway" v="trunk"/>
</way>

**Comment by mtmail** [comment]
Nominatim doesn't have such a feature.

**Reply by lhcsnelm** [reply]
thanks.  overpass api seem to have it
